alertmanager/Makefile

```diff
@@ -0,0 +1,15 @@
+APP_NAME ?= alertmanager
+GOLANG_VERSION=1.20.4
+ALERTMANAGER_VERSION=0.25.0
+BUSYBOX_VERSION=1.34.1-v1
+TAG = v$(ALERTMANAGER_VERSION)-$(DOCKER_TAG)
+IMG = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME):$(TAG)
+IMG_DOCKER_TAG = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME):$(DOCKER_TAG)
```

Comment:
You kept the versions for local development, right? The same values are set in the `envs` file and will be set by the Prow job. I am wondering if we maybe only leave the version only in one place to make upgrading more straightforward.

Reply:
Right, the Makefile is not needed for the build pipeline. Changed the build command to use the versions from the envs file.

Reply:
My attempt to "source" the envs file in the Makefile did not work reliably. I'm afraid there is no other simple solution that keeping this duplication.